OKTA-447961 Merge the OIN building/submission guides for OIDC integrations (#4146)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/create.md

4. Select **SAML 2.0** in the **Sign-in method** section.
1. Click **Next**.
1. On the **General Settings** tab, enter a name for your integration and optionally upload a logo. You can also choose to hide the integration from your end user's Okta dashboard or mobile app. Click **Next**.

> 1. On the **Configure SAML** tab, use the SAML information that you gathered in the [preparation step](#prepare-your-integration) to configure the settings of your integration. See [Create a SAML app integrations](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-saml) in the Okta product documentation.

Reviewer: ```suggestion
1. On the **Configure SAML** tab, use the SAML information that you gathered in the [preparation step](#prepare-your-integration) to configure the settings of your integration. See [Create a SAML app integration](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-saml) in the Okta product documentation.
```

> 1. On the **Configure SAML** tab, use the SAML information that you gathered in the [preparation step](#prepare-your-integration) to configure the settings of your integration. See [Create a SAML app integrations](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-saml) in the Okta product documentation.

Author: Thanks! Typo. It should be "Create SAML app integrations".

> 1. On the **Configure SAML** tab, use the SAML information that you gathered in the [preparation step](#prepare-your-integration) to configure the settings of your integration. See [Create a SAML app integrations](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-saml) in the Okta product documentation.

Author: corrected :-)

    * You can preview the generated SAML assertion by clicking **Preview the SAML Assertion** in Section B.
    * Click **Next**.
1. In the final creation step, the **Feedback** tab helps Okta to understand how you want to position this application.
    * If you're only creating an internal SAML integration:
        1. Select **I'm an Okta customer adding an internal app**. More check boxes and fields appear.

>            * Select the **App type** if you don't want the integration to be released publicly. If you check this box, you don't need to enter any further information.

Reviewer: ```suggestion
           * Select the **App type** if you don't want the integration released publicly. If you select this box, you don't need to enter any further information.
```

>            * Select the **Contact app vendor** check box if Okta needs to contact you to enable SAML for the integration. If you select this check box, you need to provide further general information about your integration to the Okta OIN team.

Reviewer: ```suggestion
           * Select the **Contact app vendor** checkbox if Okta needs to contact you to enable SAML for the integration. If you select this checkbox, you need to provide further general information about your integration to the Okta OIN team.
```
